share/spice/git_book_status/content.handlebars

```diff
@@ -0,0 +1,16 @@
+<h3 class="gbstatus--header text--primary">
+    {{! Check if the status is either good, minor or major }}
+    {{#GitbookStatus_indicator status "none"}}
+        The status of GitBook is <span class="gbstatus--good">good</span>.
```

Comment:
Please remove the trailing periods. We don't use periods in our titles/subtitlts